modules/data.atmosphere/R/download.NARR_thredds.R

```diff
@@ -0,0 +1,260 @@
+#' Retrieve NARR data using thredds
+#'
+#' @param start_date Start date for meteorology
+#' @param end_date End date for meteorology
+#' @param lat.in Latitude coordinate
+#' @param lon.in Longitude coordinate
+#' @param progress Whether or not to show a progress bar (default = `TRUE`)
+#' @return `tibble` containing time series of NARR data for the given site
+#' @author Alexey Shiklomanov
+#' @examples
+#' dat <- get_NARR_thredds("2008-01-01", "2008-01-15", 43.3724, -89.9071)
+get_NARR_thredds <- function(start_date, end_date, lat.in, lon.in,
```

Comment:
To be consistent with met.process functions , it should be named download.NARR_threads (like the name of the filename), no?  Or is the idea, that download.NARR would call this function? If not, then an outfolder needs to be specified download.XXX <- function(outfolder, start_date, end_date, site_id, lat.in, lon.in, overwrite = FALSE, verbose = FALSE) and a "results" array needs to be returned.